ref/README.md

```diff
@@ -36,15 +36,17 @@ module (e.g. `module load git`).  If you are using a system that uses modules, u
 Make sure the files in `data/inputs` are NetCDF data files (not text) before proceeding to
 the build step.
```

Comment:
I would add some more info.  The git-lfs (large file system) is used to add large files to a repository.  If you find the files in "data/inputs" to be small, likely git-lfs is not installed.

Reply:
The problem with that is that "small" is subjective and isn't the determining factor here.  The determining factor is whether or not the files are NetCDF files, and I think that is what should be verified. At this point in the document we've already clearly stated that git-lfs needs to be installed, and we've linked to it so users can find out what it does if they want to. Perhaps something like this...

....
Make sure the files in `data/inputs` are NetCDF data files (not text) before proceeding to the build step. A simple way to do that is with the `file` command as shown below:

```
$ file data/inputs/*
data/inputs/c_sw_12x24.nc: NetCDF Data Format data
data/inputs/c_sw_24x24.nc: NetCDF Data Format data
data/inputs/c_sw_48x24.nc: NetCDF Data Format data
data/inputs/c_sw_48x48.nc: NetCDF Data Format data
```
....

We've already provided a link to git-lfs in the prerequisites section for anyone who needs/wants to find out what it does.  I think we should keep this document focused on c_sw, and not add documentation and explanations for other basic tools.

Reply:
I agree, we're not in the business here of explaining get-lfs.  What I was thinking about was asking the user how to determine if the files were netCDF of not.  There was no help to the user to determine that.  So I was trying, with the file size, to help the user determine if the files are netCDF or not.  By adding the "file" explanation helps the user to determine if the files are netCDF or not.  That is the better way to go.

Reply:
This all sounds really good. I added the changes to the PR. Chris we did all of this when I was first getting the kernel to run on my Mac, so it makes sense to include the process of verifying the files the user has pulled from this repository are indeed NetCDF files using the `file` command, and not text. 

While on this topic, what are you thoughts about changing the files that are currently in the repo into "netCDF-4" format instead of "Classic" format?